scanners/amass/parser/e2e/amass.test.js

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2020 iteratec GmbH
```

Comment:
I think the e2e directory should be directly located in the `amass/` directory not in the `amass/parser/` directory as the point of a e2e test is to test it all together not just the parser.